src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -71,18 +73,39 @@ public void AddComponent(Type interfaceType, IIdentifiedComponent component)
         }
 
         /// <summary>
-        /// Gets list of available templates, if <paramref name="filters"/> are provided returns only matching templates.
+        /// Gets list of all available templates.
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns>The list of all available templates.</returns>
+        public Task<IReadOnlyList<ITemplateInfo>> GetTemplatesAsync(CancellationToken cancellationToken)
+        {
+            return _templatePackagesManager.GetTemplatesAsync(cancellationToken);
+        }
+
+        /// <summary>
+        /// Gets list of available templates, if <paramref name="filters"/> is provided returns only matching templates.
         /// </summary>
         /// <param name="filters">List of filters to apply. See <see cref="WellKnownSearchFilters"/> for predefined filters.</param>
         /// <param name="exactMatchesOnly">
         /// true: templates should match all filters; false: templates should match any filter.
         /// </param>
         /// <param name="cancellationToken"></param>
         /// <returns>Filtered list of available templates with details on the applied filters matches.</returns>
-        public Task<IReadOnlyList<ITemplateMatchInfo>> GetTemplatesAsync(IEnumerable<Func<ITemplateInfo, MatchInfo?>>? filters = null, bool exactMatchesOnly = true, CancellationToken cancellationToken = default)
+        public Task<IReadOnlyList<ITemplateMatchInfo>> GetTemplatesAsync(IEnumerable<Func<ITemplateInfo, MatchInfo?>> filters, bool exactMatchesOnly = true, CancellationToken cancellationToken = default)
```

Comment:
also not very happy here about two overloads returning different type. 
On one side, having one overload as it was before is overkill for those who just wants to get all templates, as we don't need `ITemplateMatchInfo` in this case. That's why I added the overload above.
On other side, having two methods with same name returning different lists is not nice too,

Reply:
I don't care too much, both is fine with me. Lets have same name, because we did same thing in TemplatePackageManager :)